Update KV limits with Free plan info (#2839)
cloudflare/cloudflare-docs: products/workers/src/content/platform/limits.md

Workers on the Bundled Usage Model are intended for use cases below 50 ms. Bundled Workers limits are based on CPU time, rather than [duration](#duration). This means that the time limit does not include the time a script is waiting for responses from network calls. The billing model for Bundled Workers is based on requests that exceed the included number of requests on the Paid plan. Learn more about [Usage Model pricing](/platform/pricing#usage-models).

> <Aside type="note" header="No limit* for duration">

Reviewer: ```suggestion
<Aside type="note" header="No limit for duration">
```

<TableWrap>

| Feature                               | Free                  | Paid       |
| ------------------------------------- | --------------------- | ---------- |
| [Reads/second](#kv)                   | 100,000 reads per day | unlimited  |
| [Writes/second (different keys)](#kv) | 1000 reads per day    | unlimited  |
| [Writes/second (same key)](#kv)       | 1                     | 1          |
| [Operations/worker invocation](#kv)   | 1000                  | 1000       |
| [Namespaces](#kv)                     | 100                   | 100        |
| [Keys/namespace](#kv)                 | 1 GB                  | unlimited  |
| [Key size](#kv)                       | 512 bytes             | 512 bytes  |
| [Key metadata](#kv)                   | 1024 bytes            | 1024 bytes |
| [Value size](#kv)                     | 25 MiB                | 25 MiB     |

</TableWrap>

<Aside type ="note" header="Free vs. Paid plan pricing">

> Please view our [KV pricing](/platform/pricing#workers-kv) to see the specific KV operations you're allowed under each plan with their pricing.

Reviewer: ```suggestion
Refer to [KV pricing](/platform/pricing#workers-kv) to review the specific KV operations you are allowed under each plan with their pricing.
```
